libcst/metadata/scope_provider.py

```diff
@@ -98,6 +100,65 @@ class BuiltinAssignment(BaseAssignment):
     pass
 
 
+def _get_name_for(node: Union[str, cst.CSTNode]) -> Optional[str]:
```

Comment:
Could we rename `_QualifiedNameUtil` to `_NameUtil`, and put `get_name_for` as a static method inside `_NameUtil`?

I think it'd be easier to understand if the implementations of `get_full_name_for` and `get_name_for` were adjacent.